src/components/ButtonPlain.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <Component
+    :is="!disabled && to ? 'Link' : 'button'"
+    :to="to"
+    :disabled="disabled"
+    class="button-plain"
+    @click="$emit('click', $event)"
+  >
+    <slot />
+  </Component>
+</template>
+
+<script>
+import Link from './Link.vue';
+
+export default {
+  components: { Link },
+  props: {
+    to: { type: [String, Object], default: null },
+    disabled: Boolean,
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.button-plain {
+  display: inline-block;
+  border: none;
+  background: transparent;
+  padding: 0;
+  text-decoration: none;
+  text-align: center;
+  font: inherit;
+  color: inherit;
+  cursor: pointer;
+
+  &:focus:not([data-focus-visible-added]) {
```

Comment:
This is for adding an outline when keyboard is used, it depends on https://www.npmjs.com/package/focus-visible package.